certified-connectors/The Bot Platform/apiDefinition.swagger.json

```diff
@@ -0,0 +1,393 @@
+                  "items":{
+                    "$ref":"#/definitions/UserAttribute"
+                  }
+                }
+              }
+            }
+          },
+          "400":{
+            "description":"Bad Request",
+            "schema":{
+              "type":"object",
+              "properties":{
+                
+              }
+            }
+          }
+        },
+        "summary":"Get user attributes",
+        "operationId":"PullAttributes",
+        "parameters":[
+          
+        ],
+        "description":"Get a list of attributes for a specific user in the bot",
+        "x-ms-visibility":"internal"
+      },
+      "post":{
+        "responses":{
+          "201":{
+            "description":"Created",
+            "schema":{
+              "type":"object",
+              "properties":{
+                "data":{
+                  "$ref":"#/definitions/UserAttribute"
+                }
+              }
+            }
+          }
+        },
+        "summary":"Create a user attribute",
+        "description":"Setup a new attribute in your bot",
+        "operationId":"CreateUserAttribute",
+        "parameters":[
+          {
+            "name":"body",
+            "in":"body",
+            "required":true,
+            "schema":{
+              "type":"object",
+              "properties":{
+                "data":{
+                  "type":"object",
+                  "properties":{
+                    "type":{
+                      "type":"string",
+                      "description":"type",
+                      "title":"",
+                      "default":"userattribute",
+                      "x-ms-visibility":"internal"
+                    },
+                    "attributes":{
+                      "type":"object",
+                      "properties":{
+                        "name":{
+                          "type":"string",
+                          "description":"Provide a name for your attribute",
+                          "title":"Attribute name"
+                        },
+                        "is_pii":{
+                          "type":"string",
+                          "description":"is it personally identifiable information?",
+                          "title":"is pii?",
+                          "enum":[
+                            "Yes",
+                            "No"
+                          ]
+                        }
+                      },
+                      "description":"attributes"
+                    }
+                  },
+                  "description":"data",
+                  "required":[
+                    "type"
+                  ]
+                }
+              },
+              "required":[
+                "data"
+              ]
+            }
+          }
+        ]
+      }
+    },
+    "/v1.0/users/{emailaddress}":{
+      "patch":{
+        "responses":{
+          "204":{
+            "description":"No content"
```

Comment:
@tbpsami by this definition(no schema), there will be NO return objects, can you please confirm?

Reply:
Yes, as this is a 'patch' action, we don't have a set response for this definition.